Global/Eclipse.gitignore

```diff
@@ -49,3 +49,6 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Scala IDE specific (Scala & Java development for Eclipse)
```

Comment:
Generally right.. Currently most of Scala.gitignore consists of excludes for external tools (build tools, editors, IDEs), this is not an argument to have this in it though.

But I would ask for this redundancy to cover rare corner cases, when for example one opens project without local .gitignore in it.

If you are insisting on keeping Scala.gitignore clean, I'll rework my commits a little.